core/src/main/java/org/hibernate/ogm/util/impl/Log.java

```diff
@@ -307,4 +307,7 @@
 
 	@Message(id = 90, value = "Returning multiple entities is not supported. Procedure '%1$s' expects results of type %2$s")
 	HibernateException multipleEntitiesOutputNotSupported(String procedureName, Collection<?> synchronizedQuerySpaces);
+
+	@Message(id = 91, value = "Projection and addEntity are not allowed in the same query: table <%1$s>.")
+	HibernateException addEntityNotAllowedInNativeQueriesUsingProjection( String table );
```

Comment:
I'm not sure if this message is that helpful, in the end it's not the table the problem but the query

Reply:
Unfortunately in the EmbeddedNeo4jBackendQueryResultIterator.java class we don't have the orginal query text.